sql/updates/world/3.3.5/2018_99_99_99_world_335.sql

```diff
@@ -13,8 +13,9 @@ DELETE FROM `conditions` WHERE `SourceEntry` = 17471;
 INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `Comment`) VALUES 
 (13, 0, 17471, 0, 0, 29, 1, 11197, 0, 1, 0, 0, 0, 'Stratholme - Baron Rivendare - Kill Skeleton (if alive) and Heal himself');
 
--- Stratholme - Baron Rivendare - Unholy Aura does not trigger combat
+-- Stratholme - Baron Rivendare - Unholy Aura does not trigger combat (Aura and Trigger of Aura)
 SET @SPELL_ATTR0_CU_NO_INITIAL_THREAT = 16;
-DELETE FROM `spell_custom_attr` WHERE `entry` = 17467;
+DELETE FROM `spell_custom_attr` WHERE `entry`  BETWEEN 17466 AND 17467;
```

Comment:
1 redundant space in front of `BETWEEN`, but it could just as easily be written
```sql
DELETE FROM `spell_custom_attr` WHERE `entry` IN (17466, 17467);
```

Reply:
Thanks for feedback. I will do so.